src/nvim/api/buffer.c

```diff
@@ -1339,6 +1342,79 @@ static void fix_cursor(win_T *win, linenr_T lo, linenr_T hi, linenr_T extra)
   invalidate_botline(win);
 }
 
+// Fix cursor position after replacing text
+// between (start_row, start_col) and (end_row, end_col).
+//
+// win->w_cursor.lnum is assumed to be >= start_row and <= end_row.
```

Comment:
```suggestion
/// Fix cursor position after replacing text
/// between (start_row, start_col) and (end_row, end_col).
///
/// win->w_cursor.lnum is assumed to be >= start_row and <= end_row.
```

Reply:
I applied this change. Though I'm not sure what this is for. I thought `///` is used for proper docstrings for an api. Does it make a difference for an internal function as well?

Reply:
yes, also internal functions can have docstrings (as documentation for core development)